artifactory/commands/npm/installorci.go

```diff
@@ -149,22 +143,9 @@ func (nic *NpmInstallOrCiCommand) prepareBuildInfoModule() error {
 	return nil
 }
 
-func (nic *NpmInstallOrCiCommand) runInstallOrCi() error {
-	log.Info(fmt.Sprintf("Running 'npm %s %s' command.", nic.cmdName, strings.Join(nic.npmArgs, " ")))
-	npmCmdConfig := &npmutils.NpmConfig{
-		Npm:          nic.executablePath,
-		Command:      append([]string{nic.cmdName}, nic.npmArgs...),
-		CommandFlags: nil,
-		StrWriter:    nil,
-		ErrWriter:    nil,
-	}
-
-	return errorutils.CheckError(gofrogcmd.RunCmd(npmCmdConfig))
-}
-
 func (nic *NpmInstallOrCiCommand) collectDependencies() error {
```

Comment:
The name of this function no longer represents all that it does. It no longer only collects depedencies, but it also runs the build now.

collectDependencies --> buildAndCollectDeps